moped-editor/src/queries/project.js

```diff
@@ -13,7 +13,7 @@ export const ADD_PROJECT = gql`
       fiscal_year
       capitally_funded
       moped_proj_phases {
-        phase_name
+        phase_id
```

Comment:
In this PR I learned that even if you don't explicitly list the field in the insert mutation, information included in the object being sent will still make its way into the database insertion